lib/checkNewCronJobs.js

```diff
@@ -0,0 +1,111 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const {
+  serverJobAdmins
+} = require('../userWhitelist.json');
+const {
+  getAllChangedFiles,
+  pingAndAssignUsers
+} = require('./utils');
+const CRON_REGISTRY_FILENAME = 'main_cron.py';
+const CRON_TEST_FILENAME = 'core/controllers/cron_test.py';
+const CRON_JOB_FILE = 'core/controllers/cron.py';
+const WIKI_LINK = 'https://github.com/oppia/oppia/wiki/' +
+ 'Testing-jobs-and-other-features-on-production';
+
+/**
+ * @param {import('probot').Octokit.PullsListFilesResponseItem[]} changedFiles
+ */
+const isCronFileChanged = (changedFiles) => {
+  const file = (fileObject) => fileObject.filename === CRON_JOB_FILE;
+  return changedFiles.some(file);
+};
+
+/**
+ * @param {import('probot').Octokit.PullsListFilesResponseItem[]} changedFiles
+ */
+const hasAddedTests = (changedFiles) => {
+  const file = (fileObject) =>
+    fileObject.filename === (CRON_REGISTRY_FILENAME || CRON_TEST_FILENAME);
+  return changedFiles.some(file);
+};
```

Comment:
If this function is named `hasAddedTests` why do we check for CRON registry?

Reply:
Basically the function checks if there is some changes made within two files i.e. `main_cron.py` for URL redirects and `cron_test.py` for tests ,if this returns true then it does not add the `registryReminder` msg in the comment  and if returned false then adds the msg to the comment. Should I rename this function to something else?

Reply:
Yep, the name doesn't really make sense.